shoop_tests/themes/classic_gray/test_plugins.py

```diff
@@ -0,0 +1,76 @@
+# -*- coding: utf-8 -*-
+# This file is part of Shoop.
+#
+# Copyright (c) 2012-2016, Shoop Ltd. All rights reserved.
+#
+# This source code is licensed under the AGPLv3 license found in the
+# LICENSE file in the root directory of this source tree.
+import pytest
+from bs4 import BeautifulSoup
+
+from shoop.core.models import Product, ProductCrossSell
+from shoop.testing.factories import create_product, get_default_shop
+from shoop.themes.classic_gray import plugins
+from shoop_tests.front.fixtures import get_jinja_context
+
+
+@pytest.mark.django_db
+def test_cross_sell_plugin_type():
```

Comment:
This test contains a lot of duplicate code which could be simplified with few functions.